### PR TITLE
Various2

### PR DESCRIPTION
**Pull request by lordfolken**

### CODE REVIEW

File: src/Repository/FileType.hpp [hunk 1]
```diff
@@ -14,5 +14,6 @@ enum class FileType : uint8_t {
   FLARMNET,
   IGC,
   RASP,
-  XCI
+  XCI,
+  TASK
```
Comment (MaxKellermann) [inline src/Repository/FileType.hpp:18]: Trailing comma!

Comment (lordfolken) [reply src/Repository/FileType.hpp:18]: To extend the enum I need to add that comma. I believe this to be correct.

Comment (MaxKellermann) [reply src/Repository/FileType.hpp:18]: Of course, but I mean you should add a trailing comma! The last guy (who added `XCI`) forgot it and now you had to modify an unrelated line.

Comment (lordfolken) [reply src/Repository/FileType.hpp]: Got it. I split the commit in 2, one to fix the missing comma, and the second one to add the task incl. trailing comma.

**Comment by GeekGoneOld** [comment]
I am pretty sure Max will insist on a comma after “TASK”.  This format was introduced to C++ over a decade ago and is preferred as it makes adding to the list and swapping items much cleaner.

**Pull request merged**